website/client/js/controllers/rootCtrl.js

```diff
@@ -24,8 +24,10 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
     $rootScope.$on('$stateChangeSuccess',
       function(event, toState, toParams, fromState, fromParams){
         $rootScope.pageTitle = $state.current.title;
-
-        $window.scrollTo(0, 0);
+        
+        if ($state.current.name !== 'options.social.challenges.detail') {
```

Comment:
Copying my comment here, since the new commit hid it:

We should instead do something like:

```js
var IGNORE_SCROLL_PAGES = {
  'options.social.challenges.detail': true
};

// rest of file
// inside $rootScope.$on('$stateChangeSuccess',

if (!($state.current.name in IGNORE_SCROLL_PAGES)) {
  $window.scrollTo(0, 0);
}
```

This will allow us to easily add additional exceptions.